variables.tf

```
@@ -66,6 +68,34 @@ variable "unique_writer_identity" {
   default     = null
 }
 
+# FOLDER
+
+variable "folder" {
+  type        = string
+  description = "(Optional) The ID of the folder to create the sink in. If omitted, no folder logging sink is created. If provided along with `var.project`, only the project logging sink is created."
+  default     = null
+}
+
+variable "include_folder_children" {
```

```suggestion
variable "include_children" {
```

we do not need multiple variables here...